src/gluonts/mx/block/quantile_output.py

```diff
@@ -20,47 +20,95 @@
 from gluonts.mx import Tensor
 
 
+def uniform_weights(objects: list) -> List[float]:
+    """
+    Return uniform weights for a list of objects.
+
+    >>> uniform_weights(["a", "b", "c", "d"])
+    [0.25, 0.25, 0.25, 0.25]
+
+    Parameters
+    ----------
+    objects
+        Objects that need to be weighted.
+
+    Returns
+    -------
+    List[float]
+        List of weights.
+    """
+    return [1.0 / len(objects)] * len(objects)
+
+
+def crps_weights_pwl(quantile_levels: List[float]) -> List[float]:
```

Comment:
@dcmaddix as @jaheba pointed out, weights produced by this function don't sum to one in general. Only if 0 and 1 are included as quantile levels they do:

```
In [4]: sum(crps_weights_pwl([0.0, 0.1, 0.9, 0.95, 1.0]))
Out[4]: 1.0

In [5]: sum(crps_weights_pwl([0.1, 0.9, 0.95]))
Out[5]: 0.8499999999999999
```

Reply:
Yes, I think 0 and 1 should be included here since the CRPS is the integral over the quantiles from 0 to 1, which this function approximates. Best to confirm with @youngsuk0723.

Reply:
@dcmaddix is correct. @lostella The sum is 1 when to include 0, 1. Otherewise, it is less than 1, which is also similar to the way of approximaying CRPS in original MQ-CNN.